src/effects.js

```diff
@@ -20,4 +20,5 @@ export {
   takeEvery,
   takeLatest,
   throttle,
+  retry,
```

Comment:
im not yet sure if this should be treated as effect, but that's not important right now, I can easily decide about it after merging your PR in and just move the public export appropriately

Reply:
Yeah, I also don't think it classifies as an effect.